pkg/skaffold/runner/changeset.go

```diff
@@ -40,6 +41,7 @@ func (c *changeSet) AddRebuild(a *latest.Artifact) {
 	}
 	c.rebuildTracker[a.ImageName] = a
 	c.needsRebuild = append(c.needsRebuild, a)
+	c.needsRetest = true
```

Comment:
we can probably remove this since you have it in `dev.go` already.

Reply:
Removed.